Maths/FindMax.py

```diff
@@ -7,9 +7,5 @@ def find_max(nums):
       if x > max:
         max = x
     print(max)
-
-def main():
-  find_max([2, 4, 9, 7, 19, 94, 5])
-
-if __name__ == '__main__':
-  main()
+L=list(map(int,input().split()))
```

Comment:
1. Nice idea to take input from the user, but please add a message like **Enter numbers separated by commas** inside `input()` to tell the user what to do.
2. Put your code inside the **main()** method!

Reply:
If you want to input values separated by commas then put `split (',')`. split () will separate values by spaces.